lib/utils/evidence.ts

```diff
@@ -0,0 +1,25 @@
+import { Applicant } from '../../domain/HousingApi';
+import { getQuestionValue } from '../store/applicant';
+import { FormID } from './form-data';
+
+export const getRequiredDocumentsForApplication = (
+  applicant: Applicant
+): Array<string> => {
+  let documentTypes = [
```

Comment:
fun fact: you could still use `const` here. 

Because `documentTypes` itself isn't changing when you `push` to it below. Just the data held inside of it. Typescript has a handy `as const` that you can add if you want to make objects that can't be mutated but it's rarely necessary.
If you wanted to avoid the mutation all together then this would also work:

```
return  [
    'proof-of-id',
    'proof-of-address',
    'proof-of-income',
    'proof-of-savings',
   ...(employment === 'retired' ? ['proof-of-pension'] : [])
];

```

I'll let you decide which is the neater of the two.

Reply:
those strings should also be from some kind of enum in the domain model I reckon